apertium/utils.py

```diff
@@ -36,23 +38,35 @@ def execute_pipeline(inp: str, commands: List[List[str]]) -> str:
     """
     end = inp.encode()
     for command in commands:
+        lttoolbox.LtLocale.tryToSetLocale()
+        input_file = tempfile.NamedTemporaryFile('w', delete=False)
+        output_file = tempfile.NamedTemporaryFile('r', delete=False)
+        arg = command[1][1] if len(command) == 3 else ''
+        path = command[-1]
+        wrapper_flag = True
```

Comment:
```suggestion
        used_wrapper = True
```

Generally, there's a better variable name (than one that says "flag") when you're trying to use a flag.